Client/Frontend/Brave Rewards/Transfer/WalletTransferCompleteViewController.swift

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+
+class WalletTransferCompleteViewController: UIViewController, Themeable {
+    
+    private var transferCompleteView: WalletTransferCompleteView {
+        view as! WalletTransferCompleteView // swiftlint:disable:this force_cast
+    }
+    
+    override func loadView() {
+        view = WalletTransferCompleteView()
+        applyTheme(Theme.of(nil))
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationItem.hidesBackButton = true
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(tappedDone))
+        title = "Wallet Transfer"
+    }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
```

Comment:
What do you guys think about creating a BaseViewController and adding traitCollectionDidChange and maybe some other repeated functionalty inside

Reply:
Do we always have the necessary information to apply a theme based on tab from anywhere? I would prefer that if we could do it